### PR TITLE
Add kui KEP

### PR DESCRIPTION
**Pull request by starpit**
This PR adds a sig-cli/kui KEP.

### CODE REVIEW

**Comment by starpit** [comment]
Cla updated

**Comment by starpit** [comment]
I signed it

File: keps/sig-cli/20191111-kui.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+title: KEP Template
```
Comment (sftim) [inline keps/sig-cli/20191111-kui.md]: @starpit this title should change

Comment (starpit) [reply keps/sig-cli/20191111-kui.md]: thanks for finding this. updated!

**Comment by pwittrock** [comment]
holding for @soltysh

**Pull request merged**